tests/contrib/actions/reports/test_reports.py

```diff
@@ -0,0 +1,40 @@
+try:
+    from unittest.mock import MagicMock, patch, call, mock_open
```

Comment:
You can use `pytest-mock` library (and the `mocker` fixture) as you did in `test_files.py` to keep consistency.